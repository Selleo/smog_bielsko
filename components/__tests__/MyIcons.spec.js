import 'react-native';
import React                          from 'react';

import chai, { expect }               from 'chai';
import dirtyChai                      from 'dirty-chai';

import glyphMap            from '../MyIcons';

const originalExpect = global.expect;

chai.use(dirtyChai);

describe.only('StationDataRenderer', () => {
  it('renders correctly', () => {
    originalExpect(glyphMap).toMatchSnapshot();
  });
});
