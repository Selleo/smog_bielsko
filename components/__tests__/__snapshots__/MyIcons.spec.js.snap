exports[`StationDataRenderer renders correctly 1`] = `[Function]`;
